Packs/FreshDesk/Integrations/FreshDesk/FreshDesk.py

```diff
@@ -383,7 +392,7 @@ def ticket_to_incident(ticket):
     # Incident Title
     incident['name'] = 'Freshdesk Ticket: "{}"'.format(ticket.get('subject'))
     # Incident occurrence time - the ticket's creation time
-    incident['occurred'] = ticket.get('created_at')
+    incident['occurred'] = ticket.get('updated_at')
```

Comment:
The API filters the tickets by update time and not by created time. If we take the creation time instead of the update time, the fetch incident function will not progress in the next fetch (in some cases).

for example:
`ticket1:  {
    created_time: 12.5,
   updated time: 14.5
},
`ticket2:  {
    created_time: 12.5,
   updated time: 14.5
}
`

Let say we fetch 1 incident per fetch, so the first one will be fetched and then the "created date" will be saved (12.5) for the next fetch. Then we fetch again and the API will take 12.5 as the update time so the same ticket will be fetched again.